src/System.Net.Http/src/uap/System/Net/HttpClientHandler.cs

```diff
@@ -350,36 +363,52 @@ public SslProtocols SslProtocols
 
         public HttpClientHandler()
         {
-            _rtFilter = new RTHttpBaseProtocolFilter();
-            _handlerToFilter = new HttpHandlerToFilter(_rtFilter);
+            _rtFilter = CreateFilter();
+            _rtFilterWithNoCredentials = CreateFilter();
+
+            _handlerToFilter = new HttpHandlerToFilter(_rtFilter, _rtFilterWithNoCredentials, this);
             _handlerToFilter.RequestMessageLookupKey = RequestMessageLookupKey;
             _handlerToFilter.SavedExceptionDispatchInfoLookupKey = SavedExceptionDispatchInfoLookupKey;
             _diagnosticsPipeline = new DiagnosticsHandler(_handlerToFilter);
 
             _clientCertificateOptions = ClientCertificateOption.Manual;
 
+            _useCookies = true; // deal with cookies by default.
+            _cookieContainer = new CookieContainer(); // default container used for dealing with auto-cookies.
+
+            _allowAutoRedirect = true;
+            _maxAutomaticRedirections = 50;
+
```

Comment:
It is not 10 because it was only 10 due to using auto redirection with WinRT.  And that 10 was not settable...it was hard code within WinRT.

But now that we do it ourselves,, we can have the default be the same as the rest of .NET Framework, .NET Core.